internal/schemautil/mutations.go

```diff
@@ -61,7 +94,21 @@ func normalizeIpFilter(old, new map[string]interface{}) {
 		found := false
 
 		for _, o := range oldIpFilters {
-			if n == o {
+			var comparableO interface{}
+
+			var comparableN interface{}
+
+			if fieldToWrite == "ip_filter" {
+				comparableO = o
+
+				comparableN = n
+			} else {
+				comparableO = o.(map[string]interface{})["network"]
+
+				comparableN = n.(map[string]interface{})["network"]
+			}
+
+			if comparableO == comparableN {
 				found = true
```

Comment:
I think this can be easier:
1. create a slice and a map `seen` for old values
2. iterate over old slice and append to new, and `seen`
3. iterate over new slice and if value is not in seen, then append it

Reply:
this would be good to consider for refactoring it to work this way in future